src/common/servlist.c

```diff
@@ -132,19 +117,18 @@ static const struct defaultserver def[] =
 
 	{"Dark-Tou-Net",	0},
 	{0,			"irc.d-t-net.de"},
-
-	{"DeltaAnime", 0},
-	{0,			"irc.deltaanime.net"},
+	
+#ifdef USE_OPENSSL
+	{"DosersNET", 0, 0, 0, LOGIN_SASL, 0, TRUE},
+	/* Self signed */
```

Comment:
Mmmm. I don't thing I want self-signed servers in this list.

Reply:
This server is no longer self signed.